### DISCUSSION TITLE
Stellar Quest Submission (P1U3M Gameboy edition)

### DISCUSSION THREAD

**Discussion by p1u3m** [stellar/sorobanathon#107] [General]
**This Gameboy project made for Stellar Quest Art Quest with collaboration of Rwx.quest**

> gameplay video 

https://user-images.githubusercontent.com/10558448/205477253-6367012a-ee65-4fbe-837b-e650a574fbb4.mp4

> Play directly on itch.io

https://p1u3m.itch.io/stellar-quest

or

> You can use this ROM on your emulator to play this virtual novel game 

[StellarQuest.zip](https://github.com/stellar/sorobanathon/files/10147963/StellarQuest.zip)

_this is the recommend web emu with (CRT mattias Shader)_
https://demo.emulatorjs.org/

**Comment by LorDDark6660** [comment]
Wow!

**Comment by rawritude** [comment]
This is great! If you ever want to mint a game on litemint, you can insert it as an embed in the description. It's a little controversial though cause the meta can be refreshed/not permanent. Kind of like generative dungeons on opensea was.

**Reply by p1u3m** [reply]
Thanks for the advice!

**Comment by briwylde08** [comment]
This is insane, I love it so much!

**Reply by p1u3m** [reply]
thank you!

**Comment by stellarjohn** [comment]
Hi pluem544!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Reply by p1u3m** [reply]
Thanks John, I just submit individual KYC how long does it take for the confirmation

**Reply by stellarjohn** [reply]
These are handled manually so the earliest will be sometime next week. I'll ping the manager ;0

**Reply by p1u3m** [reply]
Sorry sir, Is this KYC still in the process?

**Comment by silence48** [comment]
wow.

**Reply by p1u3m** [reply]
thanks!

**Comment by MiriamAndolini** [comment]
This is incredible!
